frontend/packages/metal3-plugin/src/components/baremetal-hosts/BareMetalHostDetails.tsx

```diff
@@ -185,9 +188,15 @@ const BareMetalHostDetails: React.FC<BareMetalHostDetailsProps> = ({
                 <dt>Bios</dt>
                 <dd>
                   <DetailPropertyList>
-                    <DetailPropertyListItem title="Version">{bios.version}</DetailPropertyListItem>
-                    <DetailPropertyListItem title="Vendor">{bios.vendor}</DetailPropertyListItem>
-                    <DetailPropertyListItem title="Date">{bios.date}</DetailPropertyListItem>
+                    <DetailPropertyListItem title="Version">
+                      {bios.version || DASH}
+                    </DetailPropertyListItem>
+                    <DetailPropertyListItem title="Vendor">
+                      {bios.vendor || DASH}
+                    </DetailPropertyListItem>
+                    <DetailPropertyListItem title="Date">
+                      {bios.date || DASH}
```

Comment:
Shouldn't this be humanized?

Reply:
out of scope for this PR, but I will verify once there's some baremetal env available

Reply:
sounds good